control/velocity_controller/src/velocity_controller_core.cpp

```diff
@@ -641,7 +650,8 @@ bool VelocityController::isEmergencyState(int closest, double target_vel) const
   return false;
 }
 
-bool VelocityController::isValidTrajectory(const autoware_planning_msgs::msg::Trajectory & traj) const
+bool VelocityController::isValidTrajectory(const autoware_planning_msgs::msg::Trajectory & traj)
+const
```

Comment:
It looks a little bit wierd.  I think it's better as below.
```
bool VelocityController::isValidTrajectory(
  const autoware_planning_msgs::msg::Trajectory & traj) const
```

Reply:
this is cause by uncursify i will fix this